src/blockdigest.cpp

```diff
@@ -16,8 +16,7 @@ std::vector<uint8_t> base_filter::outpoint_data(const COutPoint& outpoint) const
 {
     CDataStream stream(SER_NETWORK, PROTOCOL_VERSION);
     stream << outpoint;
-    std::vector<uint8_t> data(stream.begin(), stream.end());
-    return data;
+    return std::vector<uint8_t>(stream.begin(), stream.end());
```

Comment:
There's something going on here where the tx_data vector is a vector of uint8_t vectors and I need both that and regular uint8_t vectors. I.e. if I made an "append_outpoint" method which took a regular vector, I'd then no longer append directly into tx_data; I would create a new vector, append the outpoint to it, and then append that to tx_data. If I do vec-of-vecs, I now have to create a vec-of-vecs, append to it, then pull out the 0th entry for the other cases.

If I'm making no sense, please ignore. I'll look again when I'm more awake. :)